src/internal/collection/postgres_schema.go

```diff
@@ -102,6 +124,48 @@ $$ language 'plpgsql';
 	if _, err := sqlTx.ExecContext(ctx, notifyFunction); err != nil {
 		return errors.EnsureStack(err)
 	}
+
+	notificationTable := `
+create table if not exists collections.large_notifications (
+	id serial,
+	createdat timestamp with time zone default current_timestamp,
+	proto bytea
+)
+`
+
+	if _, err := sqlTx.ExecContext(ctx, notificationTable); err != nil {
+		return errors.EnsureStack(err)
+	}
+
+	notificationsIndex := "create index on collections.large_notifications (createdat);"
+	if _, err := sqlTx.Exec(notificationsIndex); err != nil {
+		return errors.EnsureStack(err)
+	}
+
+	// The sweep function makes sure that the size of the large_notifications
+	// table is bounded by deleting notifications over a certain age when a new
+	// notification is written.
```

Comment:
Also add a function to delete old notification proto data any time we write to the `large_notifications` table because unbounded growth bad.